tests/rgw/sanity_rgw_multisite.py

```diff
@@ -92,7 +92,10 @@ def run(**kw):
         set_test_env(config, primary_rgw_node)
         set_test_env(config, secondary_rgw_node)
 
-        if primary_cluster.rhcs_version.version[0] == 5:
+        ceph_version, rc = primary_rgw_node.exec_command(
```

Comment:
Can we replace `rc` with `_` if we are not using `rc` further

Reply:
Made changes